package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "sib-api-v3-typescript",
-    "version": "1.2.0",
+    "version": "6.13.5",
```

Comment:
Why is package-lock.json version different from package.json version?

Reply:
Corrected !